server/app/views/components/ToastFragment.html

```diff
@@ -43,3 +51,34 @@
     ></div>
   </th:block>
 </div>
+
+<!--/* Render the toasts used in the Review/Summary page */-->
+<div th:fragment="summaryToasts">
```

Comment:
question (non-blocking):  Does this mean that the "summaryToasts" fragment is only one of the fragments defined in `ToastFragment.html`?  If so, shouldn't the file be called `ToastFragments.html`?

Reply:
the convention I've been using is just that Fragment.html files can contain any number of fragments. I don't really want to establish a standard where we have to rename a file when adding a second fragment. I'd be open to naming all fragment files Fragments.html but don't feel strongly about it

Reply:
I think changing the name to the plural would be helpful as we switch over to Thymeleaf.  Maybe it's just me, but I wasn't expecting there to be multiple fragments in the file, so the naming would be helpful.

Reply:
Ok I filed https://github.com/civiform/civiform/issues/7739